src/unix/fs.c

```diff
@@ -1241,6 +1242,7 @@ static ssize_t uv__fs_copyfile(uv_fs_t* req) {
   off_t in_offset;
   off_t bytes_written;
   size_t bytes_chunk;
+  char* cow_buf;
```

Comment:
This is going to cause a warning on platforms other than apple, maybe declare it inside the if?

Reply:
I wrote it like that due to the style/lint rules.

Reply:
then add a `(void) cow_buf;` in the else ifdef branch pl.

Reply:
Fixing it!

Reply:
I think it is not a sin having a conditional for platform-specific variables

https://github.com/libuv/libuv/blob/bf61390769068de603e6deec8e16623efcbe761a/src/unix/stream.c#L404